test/native/test_transport_bus_can.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Unit tests of the libuavcan CAN bus headers.
+ */
+
+#include "libuavcan/uavcan.hpp"
+#include "libuavcan/bus/can.hpp"
+#include "gtest/gtest.h"
+
+namespace libuavcan
+{
+/**
+ * Verify the immutable properties of CAN FD
+ */
+TEST(CanBusTest, TypeFd)
```

Comment:
```suggestion
TEST(CanBusTest, TypeFD)
```

Reply:
There's a few more placed I did this. Let be do a s&r and update the PR.